wpovernight/woocommerce-pdf-invoices-packing-slips
# HPOS compatibility

closes #420

### How to test?

You can test HPOS now using WooCommerce version 7.1, no need to install a special version anymore. You should follow this steps: https://developer.woocommerce.com/2022/09/16/high-performance-order-storage-call-for-testing-vol-2/#download-and-install

HPOS Sync is still much under development, it might not work as expected, I also had issues with it which I posted here: https://github.com/woocommerce/woocommerce/issues/34981

Desired testing setups:

* Existing WC <=7.0 setup + WC 7.1 upgrade + HPOS **enabled** + Sync **enabled** (can have issues)
* New WC 7.1 setup + HPOS **enabled** + Sync **disabled**
* Existing WC <7.1 setup + HPOS **disabled**
* New WC 7.1 setup + HPOS **disabled**

By "existing" I mean older versions of WC prior to 7.1 with orders created in the `wp_posts` table.
